src/Dfe.PlanTech.Infrastructure.Data/CmsDbContext.cs

```diff
@@ -129,9 +129,95 @@ public CmsDbContext()
 
     public CmsDbContext(DbContextOptions<CmsDbContext> options) : base(options)
     {
-        _queryCacher = GetRequiredService<IQueryCacher>();
+        _contentfulOptions = this.GetService<ContentfulOptions>() ?? throw new MissingServiceException($"Could not find service {nameof(ContentfulOptions)}");
+        _queryCacher = this.GetService<IQueryCacher>() ?? throw new MissingServiceException($"Could not find service {nameof(IQueryCacher)}");
     }
 
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.ApplyConfigurationsFromAssembly(typeof(CmsDbContext).Assembly);
+
+        modelBuilder.Entity<RecommendationChunkContentDbEntity>()
+            .ToTable("RecommendationChunkContents", Schema);
+
+        modelBuilder.Entity<RecommendationIntroDbEntity>()
+            .ToTable("RecommendationIntros", Schema);
+
+        modelBuilder.Entity<RecommendationSectionDbEntity>()
```

Comment:
ah interesting, so this ordering does matter then?

Reply:
Yeah it does, but only if you configure the same property multiple times (in which case, the last configuration "wins"). I actually don't think _that_ PR was breaking anything, but I also want to just be sure so it's gone as well for now. Can be addressed in future